service/storage_service.cc

```diff
@@ -421,10 +421,9 @@ future<> storage_service::topology_state_load() {
                 // for the node were not in the 'normal' state yet
                 co_await update_peer_info(ip);
                 // And then amend with the info from raft
-                co_await _sys_ks.local().update_tokens(ip, rs.ring.value().tokens);
+                co_await _sys_ks.local().update_tokens(ip, id.uuid(), rs.ring.value().tokens);
                 co_await _sys_ks.local().update_peer_info(ip, "data_center", rs.datacenter);
                 co_await _sys_ks.local().update_peer_info(ip, "rack", rs.rack);
-                co_await _sys_ks.local().update_peer_info(ip, "host_id", id.uuid());
```

Comment:
This would also call update_cached_values. Which apparently does nothing. Maybe we should remove the function.

Reply:
sure